sources/engine/Stride.Engine/Rendering/Compositing/ForwardRenderer.cs

```diff
@@ -712,7 +712,7 @@ protected override void DrawCore(RenderContext context, RenderDrawContext drawCo
                     }
 
                     //draw mirror to backbuffer (if size is matching and full viewport)
-                    if (VRSettings.CopyMirror)
+                    if (VRSettings.CopyMirror && VRSettings.VRDevice.MirrorTexture != null)
```

Comment:
Was this still necessary in the end to avoid exceptions?

Reply:
Apparently OpenXR doesn't support this feature. Instead of checking the texture from the forward renderer I've added a CanMirrorTexture property to VRDevice that implementations can override to signal they don't support this feature.

Then VRSettings.CopyMirror will return false no matter what it's set to. Not sure if that's the best way to implement it but at least it's clearer than checking from the renderer